public/js/mc-answer.js

```diff
@@ -5,8 +5,6 @@ var mcAnswerCount = 4; // default number of options
 */
 var addMCAnswers = function(dom){
     mcAnswerCount++;
-    var newdiv = document.createElement('p');
-    var inputdiv = '<input type="radio" name="radbutton" value="mcans{0}" id="mcans{1}" required/><label for="mcans{2}">Is Correct</label><input type="text" name="mcans{3}" placeholder="Enter Answer Here" required style="float:left;" class="form-control"/><a onclick="$(this).parent().remove()" class="btn-floating btn-tiny waves-effect waves-light red"><i class="tiny material-icons">close</i></a>';
-    newdiv.innerHTML = inputdiv.format(mcAnswerCount,mcAnswerCount,mcAnswerCount,mcAnswerCount);
-    $('#qAnswer > div.form-group').append(newdiv);
-}
+    var inputdiv = "<div class='row'><div class='col s1'><input type='radio' name='radbutton' value='mcans{0}' id='mcans{1}' required='required'><label for='mcans{2}'></label></div><div class='input-field col s10'><input class='form-control' type='text' name='mcans{3}' placeholder='Enter Answer Here' required='required' style='float:left;'></div><div class='input-field col s1'><a class='btn-floating btn-tiny waves-effect waves-light red' onclick='$(this).parent().parent().remove()'><i class='tiny material-icons'>close</i></a></div></div>";
+    $('#qAnswer > div.form-group.col.s12').append(inputdiv.format(mcAnswerCount,mcAnswerCount,mcAnswerCount,mcAnswerCount));
```

Comment:
I changed the format function to take an array. so now all you have to do is use {0} for all of them, then pass [mcAnswerCount] to the format function